File: app/src/main/java/com/vikingsen/cheesedemo/model/data/price/Price.java
package com.vikingsen.cheesedemo.model.data.price;


public class Price {
    private final long cheeseId;
    private final double price;

    Price(long cheeseId, double price) {
        this.cheeseId = cheeseId;
        this.price = price;
    }

    public long getCheeseId() {
        return cheeseId;
    }

    public double getPrice() {
        return price;
    }
}

File: cheese-shop-ws/src/main/kotlin/com/vikingsen/model/database/price/Price.kt
package com.vikingsen.model.database.price

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Entity
data class Price(
        var cheeseId: Long = 0,
        var price: Double = 0.0,
        @Id @GeneratedValue(strategy = GenerationType.AUTO) var id: Long = 0
)